packages/plugins/plugin-table/src/components/Table/Table.tsx

```diff
@@ -192,36 +189,15 @@ const TableMain = forwardRef<TableController, TableMainProps>(({ model }, forwar
           limitRows={model?.getRowCount() ?? 0}
           limitColumns={model?.table.view?.fields?.length ?? 0}
```

Comment:
i think bette to extract this to a local variable: `numColumns`.
(Q: doesn't this need to be +1 for the fixed column?)

Reply:
I think the limit rows and columns is for the central grid plane.